include/sbg_driver/sbg_matrix3.h

```diff
@@ -0,0 +1,127 @@
+#ifndef SBG_MATRIX_3_H
+#define SBG_MATRIX_3_H
+
+// Sbg headers
+#include <sbgDefines.h>
```

Comment:
Add at the top of the file the SBG copyrights.
Also explain that ROS uses Eigen for mathematical computations so we implement a very basic matrix class to avoid Eigen dependencies.